recipe/meta.yaml

```diff
@@ -35,11 +34,6 @@ requirements:
     - tinyxml
     - console_bridge
   run:
-    - libignition-math6
-    - urdfdom
-    - boost-cpp
```

Comment:
`boost-cpp` does not have `run_exports` (see https://github.com/conda-forge/boost-cpp-feedstock/pull/82). However, sdformat9 probably does not depend on boost at all (see https://github.com/osrf/homebrew-simulation/blob/master/Formula/sdformat9.rb), so probably we can remove it also from the host dependencies.

Reply:
I did this modification in https://github.com/conda-forge/libsdformat-feedstock/pull/45/commits/f55676dacbedc158255d8bb8ebfea807bc67ec4a .

Reply:
Thanks for the catch @traversaro!